### PR TITLE
Unsafe: refactor UNSAFE_ methods from /market/ and others

### PR DESCRIPTION
**Pull request by gitstart-augur**
#4342

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). ⚡️
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/r9zh037**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/market/components/market-view/market-view.tsx [hunk 1]
```diff
@@ -179,42 +179,23 @@ export default class MarketView extends Component<
...
       isConnected,
       marketId,
       isMarketLoading,
       closeMarketLoadingModal,
-      tradingTutorial,
-      updateModal,
-    } = this.props;
-    this.tradingTutorialWidthCheck();
-
-    if (tradingTutorial) {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market/components/market-view/market-view.tsx:193]: why was this code removed?

File: packages/augur-ui/src/modules/filter-sort/components/filter-search.tsx [hunk 2]
```diff
@@ -48,22 +51,16 @@ export default class FilterSearch extends Component<
     this.timeout = null;
   }
 
-  UNSAFE_componentWillMount() {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/filter-sort/components/filter-search.tsx:51]: why didn't `UNSAFE_componentWillMount` get removed and not converted?

File: packages/augur-ui/src/modules/market/components/market-view/market-view.tsx (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/market/components/market-view/market-view.tsx:193]: I'll fix

File: packages/augur-ui/src/modules/filter-sort/components/filter-search.tsx (hunk 2 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/filter-sort/components/filter-search.tsx:51]: after testing, looks like componentWillMount can be removed.

**Review by bthaile — APPROVED** [review]
fixed an issue, lgtm now

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). ⚡️
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/m96h167**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

**Pull request merged**